MiniSim/Views/Preferences.swift

```diff
@@ -11,8 +11,24 @@ import KeyboardShortcuts
 import LaunchAtLogin
 
 struct Preferences: View {
+    @State var menuImageSelected: String = "iphone"
```

Comment:
I'm using `@AppStorage` in `MiniSim/Views/Onboarding/SetupView.swift` and it works great with Picker so I'm not sure whats the issue here. Can you check out that file?

Reply:
@okwasniewski Sorry for the long delay, I've been a bit busy lately. SetupView works because `enableiOSSimulators` and `enableAndroidSimulators` are changed before the menu setup is called. With `menuImageSelected`, it's a bit different, because the menu needs to change on the fly, as a `@State` would. I have seen examples of `@AppStorage` acting this way, but it isn't recurring for me.

For example, if I call `enableAndroidSimulators` in this way:
```swift
struct Preferences: View {
    ...
    @AppStorage(UserDefaults.Keys.enableAndroidEmulators, store: .standard) var enableAndroidEmulators: Bool = true

    var body: some View {
        Settings.Container(contentWidth: 400) {
                ...
                Settings.Section(title: "Enable Android Emulators:") {
                        Toggle(isOn: $enableAndroidEmulators) {
                        Text("Enable Android Emulators")
                }
                ...
 ```
 it doesn't mutate whilst it's open, because at least in my case, `@AppStorage` doesn't act like State.